am-lib/src/main/resources/db/migration/migrationAks/V19__load_data.sql

```diff
@@ -7,3 +7,6 @@ insert into resources (service_name, resource_type, resource_name) values ('cmc-
 insert into roles (role_name, role_type, security_classification, access_type) values ('caseworker-test', cast('RESOURCE' as role_type),cast('PUBLIC' as security_classification), cast('ROLE_BASED' as access_type))
  on conflict on constraint roles_pkey do update set role_type =cast('RESOURCE' as role_type), security_classification = cast('PUBLIC' as security_classification), access_type = cast('ROLE_BASED' as access_type);
 
+insert into roles (role_name, role_type, security_classification, access_type) values ('caseworker-test1', cast('IDAM' as role_type),cast('PUBLIC' as security_classification), cast('ROLE_BASED' as access_type))
+ on conflict on constraint roles_pkey do update set role_type =cast('RESOURCE' as role_type), security_classification = cast('PUBLIC' as security_classification), access_type = cast('ROLE_BASED' as access_type);
+
```

Comment:
update statement on conflict seems to be wrong... should set role_type = IDAM instead of resource